### PR TITLE
Fix #1150: Non-personalized operations and operation claim

### LINKED ISSUE #1150: Non-personalized operations and operation claim
For QR code login functionality following changes are needed:
- Allow to create an operation with `null` userId value.
- Claim of operation during operation retrieval.

Add Liquibase script for DB migration and update migration guide.

### PR DESCRIPTION
**Pull request by romanstrobl**

### CODE REVIEW

**Review by petrdvorak — APPROVED** [review]
Looks OK, thank you for doing the heavy lifting on this!

**Review by petrdvorak — DISMISSED** [review]

File: docs/PowerAuth-Server-1.6.0.md [hunk 1]
```diff
@@ -4,6 +4,23 @@ This guide contains instructions for migration from PowerAuth Server version `1.
 
 ## Database Changes
 
+### Allow Non-personalized Operations
```
Comment (banterCZ) [inline docs/PowerAuth-Server-1.6.0.md:7]: I have noticed that this file is not linked from `Migration-Instructions.md` yet.

File: docs/PowerAuth-Server-1.6.0.md [hunk 2]
```diff
@@ -4,6 +4,23 @@ This guide contains instructions for migration from PowerAuth Server version `1.
...
+
+The column `user_id` in table `pa_operation` is nullable now.
+
+#### PostgreSQL
```
Comment (banterCZ) [inline docs/PowerAuth-Server-1.6.0.md:11]: IMHO the agreement was to only describe the change but not write SQL manually anymore. cc @zcgandcomp

File: powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationDetailRequest.java [hunk 3]
```diff
@@ -29,4 +29,9 @@
 public class OperationDetailRequest {
 
     private String operationId;
+
+    /**
+     * Optional user identifier of the user who is requesting the operation.
+     */
+    private String userId;
```
Comment (banterCZ) [inline powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationDetailRequest.java:36]: Missing in `WebServices-Methods.md`.

**Review by banterCZ — APPROVED** [review]
Looks good.

File: docs/PowerAuth-Server-1.6.0.md (hunk 2 above)
Comment (romanstrobl) [reply docs/PowerAuth-Server-1.6.0.md:11]: I checked the syntax and it should be correct. I will keep the migration guide manually written for now, we can update it next time.

File: powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationDetailRequest.java (hunk 3 above)
Comment (romanstrobl) [reply powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationDetailRequest.java:36]: Added.

**Review by banterCZ — APPROVED** [review]

**Pull request merged**